src/Compilers/Core/Portable/Compilation/IExpression.cs

```diff
@@ -841,10 +841,36 @@ public interface IConversionExpression : IHasOperatorMethodExpression
         /// Kind of conversion.
         /// </summary>
         ConversionKind ConversionKind { get; }
+
+        /// <summary>
+        /// Returns true if the conversion is an identity conversion.
+        /// </summary>
+        bool IsIdentity { get; }
+
+        /// <summary>
+        /// Returns true if the conversion is an implicit ('widening' in VB) conversion.
+        /// </summary>
+        bool IsImplicit { get; }
```

Comment:
It feels that this member can be a source of confusion. It is not an opposite to the already existing property IsExplicit. Both can return true, but the names might suggest otherwise.